app/components/Texting/Texting.scss

```diff
@@ -0,0 +1,175 @@
+@import '../../styles/utils/_helpers.scss';
+
+.overlay {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 70px;
+  background: rgba(0, 0, 0, 0.2);
+  @media screen and (max-width: 900px ){
+    top: 50px;
+  }
+}
+
+.content {
+  position: relative;
+  width: 500px;
+  height: 660px;
+  background: #FFFFFF;
+  margin: 35px auto;
+  padding: 30px;
+  box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+  box-sizing: border-box;
+  border-radius: 8px;
+
+  @media screen and (max-width: 900px ) {
+    width: 100%;
+    height: 100%;
+    font-size: 14px;
+    padding: 20px;
+    margin: 0 auto;
+  }
+}
+
+.closeButton {
+  background: none;
+  border: 0;
+  cursor: pointer;
+  display: inline-block;
+  padding: 0;
+  position: absolute;
+  right: 20px;
+  top: 20px;
+}
+
+.title{
```

Comment:
Separately, I think you need to set the font color, since the color that is showing up in the actual web app is a dark gray (`#30383A`), but in the Figma design, it should be an absolute black (`#000000`).